sdcm/sct_config.py

```diff
@@ -171,7 +171,16 @@ class SCTConfiguration(dict):
                   Test duration (min). Parameter used to keep instances produced by tests
                   and for jenkins pipeline timeout and TimoutThread.
              """),
-
+        dict(name="prepare_duration", env="SCT_PREPARE_DURATION", type=int,
+             help="""
+                  Prepare test duration (min). Parameter used to run prepare dataset before
```

Comment:
this description is misleading. Can you rephrase it?

Reply:
@soyacz , can you check, is it become more clear?